src/sonic-yang-models/yang-models/sonic-port.yang

```diff
@@ -161,6 +161,13 @@ module sonic-port{
 					}
 				}
 
+				leaf dom_polling {
+
+					type string {
+						pattern "enabled|disabled";
```

Comment:
this attribute is not required. What is the default value if missing?

Reply:
Is the default value backward compatible?

Reply:
The default value is `enabled` and is backward compatible.